.github/workflows/ci.yml

```
@@ -0,0 +1,112 @@
+            xmake:
+              - 'xmake.lua'
+              - 'src/**'
+              - 'include/**'
+              - 'tests/**'
+              - 'config/**'
+              - '.github/workflows/test-xmake.yml'
+
+  call-format:
+    needs: changes
+    if: ${{ needs.changes.outputs.format == 'true' }}
+    uses: ./.github/workflows/format.yml
```

actionlint (1.7.9)

could not read reusable workflow file for "./.github/workflows/format.yml": open /home/jailuser/git/.github/workflows/format.yml: no such file or directory

(workflow-call)